webapp-django/crashstats/base/templatetags/jinja_helpers.py

```diff
@@ -107,7 +107,9 @@ def is_dangerous_cpu(cpu_info):
     # for ease of find by users.
     return (
         cpu_info.startswith('AuthenticAMD family 20 model 1') or
-        cpu_info.startswith('AuthenticAMD family 20 model 2')
+        cpu_info.startswith('AuthenticAMD family 20 model 2') or
+        cpu_info.startswith('family 20 model 1') or
+        cpu_info.startswith('family 20 model 2')
```

Comment:
Since the new strings are pretty vague, I'm kind of loathe to do this. I think we should additionally constrain them by `cpu_arch == 'amd64'`.

Reply:
currently there are no apparent collisions at least: https://crash-stats.mozilla.com/search/?cpu_info=~family%2020%20model%201&cpu_info=~family%2020%20model%202&date=%3E%3D2017-05-02T18%3A19%3A18.000Z&_facets=cpu_info#facet-cpu_info

Reply:
It's nice that there are no collisions, but things change over time and we can't guarantee there won't be collisions and this is the kind of change where if there were, it'd probably go unnoticed for some time. Meanwhile, we'd be ignoring those crash reports because we incorrectly think they have buggy CPUs.

I think it's prudent to tie this explicitly to the `cpu_arch`.